CHANGELOG.md

```diff
@@ -21,3 +21,4 @@ and this project adheres to
 - feat: add deployer account and deploy fee to Madara genesis
 - chore: bump madara image
 - dev: rename crate conformance-test-utils to hive-utils
+- ci: use madara binary for benchmark CI
```

Comment:
i always get mixed up, are we adding new changelog stuff from top to bottom or bottom to top
i think ssj is doing opposite @enitrat

Reply:
ah, I was doing it bottom to top (more recent)

Reply:
agreed! i think that's how it works?

Reply:
In this repo, I think it top to bottom, because the topmost of one is -> "- ci: add `CHANGELOG.md` and enforce it is edited for each PR on `main`"

Reply:
Madara seems to be doing top to bottom as well -> https://github.com/keep-starknet-strange/madara/pull/1015/files

Reply:
https://devm.io/programming/dos-donts-keeping-changelog-147373

checking here

Reply:
All changes of the same type should be grouped together for reading comprehension. If you throw a deprecated feature in the middle of a bunch of fixed ones, it’s likely to be missed.
Provide links if possible! It saves time from googling and it makes sure everyone is on the right page for version changes.
The newest and latest version should be at the top of the page. Think of your twitter feed.
Document the date! People need to know when these updates were released.
Do you use semantic versioning? Then tell people.